js/index.js

```diff
@@ -16,4 +16,132 @@ for (i= 0; i < skills.length; i++){
     let skill = document.createElement('li');
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
-}
\ No newline at end of file
+}
+
+
+
+//handle message form submit
+let messageForm = document.getElementsByName("leave_message");
+
+console.log(messageForm[0], "this is the form");
+
+
+
+messageForm[0].addEventListener("submit", function(event){
```

Comment:
When you only want to get one particular element, it's a little better to use the ID. With `getElementsByName`, there could be multiple matches. On a larger project this could be confusing.

Reply:
The lesson asked for me to get the element by name.